recipes/hole-filling-liepa/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "hole-filling-liepa" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  url: https://github.com/russelmann/hole-filling-liepa/archive/v{{ version }}.tar.gz
+  sha256: c73a65bd39b30801f5c85f7245de0f1ae8458d6eb78d657f63d8976d63cf4a47
+
+build:
+  number: 0
+  skip: true  # [py<38]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - make
+    - pip
+    - pybind11
+    - eigen
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - numpy
```

```suggestion
    - {{ pin_compatible('numpy') }}
```
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

If you actually link your C++ code against the NumPy API, then you need to pin a compatible version.

---

I was too quick to commit this. There is no linking to NumPy, so I will undo this change.